### ISSUE TITLE
Create saved list from import

### ISSUE THREAD

**Issue by JimKillock** [openrightsgroup/cmp-issues#243]
Hi Dan,

If it is easy, could we have a feature to create a saved list from import in the admin section?

We have 4-5 lists we would ideally add to Blocked. If this is easier than asking you to ad the data directly to the database, then the feature would be handy.

**Comment by dantheta** [comment]
The savedlists admin screen now has an "Add/upload list" button.  The linked form allows a new list to be created.  The form accepts a list name and a file upload.

The file should be plain text, one URL per line.  Uploaded URLs are normalized according to the normal URL rules.

The list is initially created as non-public and non-frontpage, though this can be toggled after creation.

If the list is public or frontpage, it will be checked weekly in the usual way.  The recheck list feature can also be used to get the status of each of the sites.

If an uploaded URL is not known to the system, it will automatically be added to the monitoring system when the list is checked.  The title field may not be automatically populated though.  I'll need to check when more of them are created.

**Comment by edjw** [comment]
Thanks Dan. I've set up some of those now. This is an example: https://www.blocked.org.uk/list/2019_report_all_charities

The import worked fine. The title field isn't populated. And the labels for 'Unblocked' and 'Reported' aren't there either

**Comment by dantheta** [comment]
Reported and unblocked will be filled in when the refresh runs overnight (or Sunday morning)

Title I'm not sure ... I'll need to check.

On 21 March 2019 16:56:56 GMT, edjw <notifications@github.com> wrote:
>Thanks Dan. I've set up some of those now. This is an example:
>https://www.blocked.org.uk/list/2019_report_all_charities
>
>The import worked fine. The title field isn't populated. And the labels
>for 'Unblocked' and 'Reported' aren't there either
>
>-- 
>You are receiving this because you were assigned.
>Reply to this email directly or view it on GitHub:
>https://github.com/openrightsgroup/cmp-issues/issues/243#issuecomment-475314897

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by dantheta** [comment]
Titles aren't currently added on import, as these are normally imported from the search engine.  I'll add a new ticket to get the title for a URL copied when the weekly refresh is happening.
